src/ke.cpp

```diff
@@ -779,3 +824,51 @@ KeReadStateTimer(_In_ PKTIMER timer)
 }
 
 #pragma endregion timers
+
+#pragma region events
+USERSIM_API
+void
+KeInitializeEvent(_Out_ PKEVENT event, _In_ EVENT_TYPE type, _In_ BOOLEAN initial_state)
+{
+    event->signalled = initial_state ? 1 : 0;
```

Comment:
Is this field only ever 0 or 1?  If so, can we use FALSE and TRUE instead of 0 and 1?  (here, line 852, etc.)